opentelemetry-otlp/examples/basic-otlp-http/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 once_cell = { workspace = true }
 opentelemetry = { path = "../../../opentelemetry" }
 opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio", "metrics", "logs"] }
-opentelemetry-otlp = { path = "../..", features = ["http-proto", "reqwest-client", "logs"] }
+opentelemetry-otlp = { path = "../..", features = ["http-proto", "http-json", "reqwest-client", "logs"] }
```

Comment:
@ramgdev Which format would be used by default? Do we need to update the README.md to specify how to switch the format?

Reply:
@lalitb please see the latest commit. The example shows how to specify the protocol when creating the exporters. We wanted to encourage clients to be explicit about their protocol choice rather than relying on the default. Hope this serves that purpose.

Reply:
Thanks @ramgdev, this provides clarity. I was also considering another option: introducing a feature flag in the Cargo.toml for this example. This would allow users to select the protocol when running the example, similar to how #1861 introduces a feature flag to select the HTTP client (either hyper or reqwest).

Reply:
The example is using a debug exporter. So, even if we offer a feature flag to change the protocol, they won't see any difference when they run it. Plus, it is easy enough to modify the code to change the endpoint and/or protocol if they really want. So, I'm thinking this is in a good state to illustrate the sdk usage.